src/app/course/course-page/course-page.component.spec.ts

```diff
@@ -22,4 +61,34 @@ describe('CoursePageComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+});
+
+describe('CourseItemTestComponent', () => {
+  let courseItem: CourseItemTestComponent;
+  let fixture: ComponentFixture<CourseItemTestComponent>;
+  let course: ICourse;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ CourseItemTestComponent ],
+      schemas: [ CUSTOM_ELEMENTS_SCHEMA ]
+    }).compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(CourseItemTestComponent);
+    courseItem = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('emit id from the course', () => {
+    course = COURSE;
```

Comment:
This test isn't valid. You should test if passed `deleteCourse` emmiter will emit id of passed componet and it should be part of ` src/app/course/course-item/course-item.component.spec.ts` test file

Reply:
Test removed.